lxd-p2c/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/lxc/lxd/shared/version"
+)
+
+type cmdGlobal struct {
+	flagVersion bool
+	flagHelp    bool
+}
+
+func main() {
+	app := &cobra.Command{}
+	app.Use = "lxd-p2c"
```

Comment:
Tbh, not the nicest name. :) Something like `lxd-containerize` or something may be better but whatever. :)

Reply:
Yeah, I don't like naming stuff but there was a lot of traction for P2V in the past so when you say P2C the right people seem to react and understand the name somehow :)